psi4/src/psi4/libmints/oeprop.cc

```diff
@@ -863,7 +863,12 @@ bool from_string(T& t,
 void OEProp::compute()
 {
 
-    outfile->Printf( "\nProperties computed using the %s density matrix\n\n", title_.c_str());
+    
+    if (title_ == "")
```

Comment:
Could use `strcmp()` and just compare to `title_.c_str()` directly

Reply:
In general please avoid C functions, the current code is fine. There is something else missing here.

Reply:
Will keep that in mind, thanks!